_partials/keep_current.md

```diff
@@ -146,6 +146,37 @@ pyenv versions
   lewagon
 ```
 
+### Install the bootcamp packages
+
+```bash
+pip install -Ur <REQUIREMENTS_URL>
+pip install -U yapf jupyterlab seaborn plotly nbconvert xgboost statsmodels pandas-profiling dtale jupyter-resource-usage
+```
+
+Install [TensorFlow](https://www.tensorflow.org/):
+$MAC_START
+
+<details>
+    <summary>Setup for Intel chips (default)</summary>
+$MAC_END
+
+```bash
+pip install -U 'tensorflow<2.6'
```

Comment:
Same thing, it would be nice to use a <TENSORFLOW_REQUIREMENTS_URL> so that we can centralize the requirements for later updates